### PR TITLE
graph-cli: init at 0.1.7

### PR DESCRIPTION
**Pull request by leungbk**
###### Motivation for this change

Add graph-cli, a Python CLI for making graphs.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/graphics/graph-cli/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    pandas
+    matplotlib
+  ];
+
+  pythonImportsCheck = [ "graph_cli" ];
+
+  meta = with lib; {
+    description = "A CLI to create graphs from CSV files";
+    homepage = "https://github.com/mcastorina/graph-cli/";
+    license = with licenses; [ gpl3 ];
```
Comment (SuperSandro2000) [inline pkgs/tools/graphics/graph-cli/default.nix:27]: gpl3Plus or gpl3Only?

File: pkgs/tools/graphics/graph-cli/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  propagatedBuildInputs = with python3Packages; [
+    numpy
+    pandas
+    matplotlib
+  ];
+
+  pythonImportsCheck = [ "graph_cli" ];
+
+  meta = with lib; {
+    description = "A CLI to create graphs from CSV files";
```
Comment (SuperSandro2000) [inline pkgs/tools/graphics/graph-cli/default.nix:25]:
```suggestion
    description = "CLI to create graphs from CSV files";
```

File: pkgs/tools/graphics/graph-cli/default.nix (hunk 1 above)
Comment (leungbk) [reply pkgs/tools/graphics/graph-cli/default.nix]: Thanks, I changed it to `gpl3Only`.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109562` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>graph-cli</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

graph-cli:
<details>

Zero tests run by pytest got detected: 'Ran 0 tests in 0.000s'
You have the following options to fix this:
  - Tell pytest(CheckHook) where to find the tests included in the package
  - Check if the GitHub Repo contains tests but they are not shipped with Pypi. If so please switch to `fetchFromGitHub`.
  - If the Packages does not contain any tests add \'doCheck = false;'` and a `pythonImportsCheck`.

</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109562` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>graph-cli</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

graph-cli:
<details>

Zero tests run by pytest got detected: 'Ran 0 tests in 0.000s'
You have the following options to fix this:
  - Tell pytest(CheckHook) where to find the tests included in the package
  - Check if the GitHub Repo contains tests but they are not shipped with Pypi. If so please switch to `fetchFromGitHub`.
  - If the Packages does not contain any tests add \'doCheck = false;'` and a `pythonImportsCheck`.

</details>
graph-cli:
<details>

Zero tests run by pytest got detected: 'Ran 0 tests in 0.000s'
You have the following options to fix this:
  - Tell pytest(CheckHook) where to find the tests included in the package
  - Check if the GitHub Repo contains tests but they are not shipped with Pypi. If so please switch to `fetchFromGitHub`.
  - If the Packages does not contain any tests add \'doCheck = false;'` and a `pythonImportsCheck`.

</details>

**Comment by leungbk** [comment]
There don't appear to be any tests; the Makefile in the GitHub repo references a nonexistent `tests` directory. For this reason, I've disabled `doCheck` and run `pythonImportsCheck` instead.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109562` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>graph-cli</li>
</ul>
</details>

File: pkgs/tools/graphics/graph-cli/default.nix [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    sha256 = "sha256-/v9COgAjuunJ06HHl55J0moV1p4uO+N+w2QwE8tgebQ=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    numpy
+    pandas
+    matplotlib
+  ];
+
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/tools/graphics/graph-cli/default.nix:21]:
```suggestion
  # does not contain tests despite reference in Makefile
  doCheck = false;
```

**Comment by SuperSandro2000** [comment]
Please squash the last review commit into the package init commit.

**Comment by leungbk** [comment]
> Please squash the last review commit into the package init commit.

Done.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of nixpkgs-review pr 109562 run on x86_64-linux 1[1]

<details>
<summary>
1 package built:
</summary>
<ul>
<li>graph-cli</li>
</ul>
</details>

[1]: https://github.com/Mic92/nixpkgs-review

**Pull request merged**